### ISSUE TITLE
Dashboards auto-discovery with registry

### ISSUE THREAD

**Issue by alex-shpak** [firehol/netdata#1848]
Hi and thanks for great project!

It's more question rather than an issue.
I'm planing to install netdata on ~30 qemu instances and after reading these:
 - https://github.com/firehol/netdata/wiki/mynetdata-menu-item
 - https://github.com/firehol/netdata/issues/178
 - https://github.com/firehol/netdata/issues/310
 - https://github.com/firehol/netdata/issues/416
 - https://github.com/firehol/netdata/issues/1311

Didn't find answer (or didn't understand).
Basically I'm looking for some kind of auto-discovery feature. When I create new instance I would like it to appear in `my-netdata` dropdown without visiting url. So I'm wondering is there such possibility?

I imagine it like: when service generate new `MachineID` it can report itself to registry and then be displayed to user.

Sorry if duplicate

**Comment by ktsaou** [comment]
no problem.

it is duplicate of #505 . There is plenty of discussion there for this issue.
I am closing this in favour of #505. Let's discuss this there.

**Comment by alex-shpak** [comment]
Sure, thanks!
